test/Dialect/Torch/canonicalize.mlir

```diff
@@ -1328,3 +1328,18 @@ func.func @torch.aten.add.Tensor$canonicalize_literal_0d() -> !torch.vtensor<[],
     %2 = torch.aten.add.Tensor %0, %1, %int3 : !torch.vtensor<[],si64>, !torch.vtensor<[],si64>, !torch.int -> !torch.vtensor<[],si64>
     return %2 : !torch.vtensor<[],si64>
 }
+
+// CHECK-LABEL:   @torch.aten.size$copy(
+// CHECK-SAME:        %[[ARG:.*]]: !torch.vtensor<[2,3],f32>) -> !torch.list<int> {
+// CHECK:           %[[two:.*]] = torch.constant.int 2
```

Comment:
uppercase variable names for consistentcy

Reply:
Makes sense.  Thanks, fixed!